Source/GoBot/ViewDatabase.swift

```diff
@@ -232,33 +232,33 @@ class ViewDatabase {
                 db.userVersion = 8
             } else if db.userVersion == 1 {
                 try db.execute("""
-                CREATE INDEX messagekeys_key ON messagekeys(key);
-                CREATE INDEX messagekeys_id ON messagekeys(id);
-                CREATE INDEX posts_msgrefs on posts (msg_ref);
-                CREATE INDEX messages_rxseq on messages (rx_seq);
-                CREATE INDEX tangle_id on tangles (id);
-                CREATE INDEX contacts_state ON contacts (contact_id, state);
-                CREATE INDEX contacts_state_with_author ON contacts (author_id, contact_id, state);
+                CREATE INDEX IF NOT EXISTS messagekeys_key ON messagekeys(key);
+                CREATE INDEX IF NOT EXISTS messagekeys_id ON messagekeys(id);
+                CREATE INDEX IF NOT EXISTS posts_msgrefs on posts (msg_ref);
+                CREATE INDEX IF NOT EXISTS messages_rxseq on messages (rx_seq);
+                CREATE INDEX IF NOT EXISTS tangle_id on tangles (id);
+                CREATE INDEX IF NOT EXISTS contacts_state ON contacts (contact_id, state);
+                CREATE INDEX IF NOT EXISTS contacts_state_with_author ON contacts (author_id, contact_id, state);
                 -- add new column to posts and migrate existing data
                 ALTER TABLE posts ADD is_root boolean default false;
-                CREATE INDEX IF NOT EXISTS posts_roots on posts (is_root);
+                CREATE INDEX IF NOT EXISTS IF NOT EXISTS posts_roots on posts (is_root);
```

Comment:
```suggestion
                CREATE INDEX IF NOT EXISTS posts_roots on posts (is_root);
```
looks like there's a duplicate `IF NOT EXISTS` on this line?